### ISSUE TITLE
[UI] New submission button is unclear in multi-journal/press context

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#950]
![new-submission-multi-context](https://cloud.githubusercontent.com/assets/2306629/11659845/39c45542-9dc2-11e5-80db-b1e4b8cd3ec4.png)

1. No indication that you're starting a new submission.
2. Select dropdowns that take action when an option is selected are not accessible, because keyboard users can not access options beyond the first.

@asmecher Would it be terrible if we just required people to make a submission in the current context, and required them to switch journal/press context if they wanted to make a submission in a different context?

**Comment by asmecher** [comment]
@NateWr, we originally planned for the dashboard to be journal (context) agnostic, i.e. not to care whether or not you were in a specific context. When in the "index" context, the dashboard was to list all submissions, regardless of press or journal. But I don't think that approach has been well maintained, and might not be advisable anyway -- mixing submissions from different presses/journals is pretty weird.

I'd suggest approaching it as you suggest. The context switcher on the dashboard, along with the change to make it go directly to the dashboard of the chosen context, resolves this better than a hopeless jumble of content in grids, IMO.

**Comment by NateWr** [comment]
Ok sounds good. I can see the utility of an "all journals/presses" view for administrators or copyeditors. But we can explore that again down the road. It's not mission-critical and I suspect we'll get lots of useful feedback on the dashboard after releases that may help us better design these processes.

Are tasks "global"?

**Comment by beghelli** [comment]
@NateWr yes, tasks are global.

**Comment by NateWr** [comment]
@asmecher The grid in [this comment](https://github.com/pkp/pkp-lib/issues/955#issuecomment-164725499) has a "Presses" column that throws the grid alignment off. I'm guessing this is the "global dashboard" thing that you talked about removing.

Do you want me to flag these up where I see them or is it already on your radar?

**Comment by asmecher** [comment]
That's correct, I'll be removing that shortly. It's supposed to display only when you're viewing the dashboard site-wide. But of course it won't do that once I've removed it :)

**Comment by asmecher** [comment]
I removed support for multi-context dashboards in issue #955.
